sdk/python/feast/infra/online_stores/contrib/hbase_online_store/README.md

```diff
@@ -0,0 +1,80 @@
+# Hbase Online Store
+Hbase is not included in current [Feast](https://github.com/feast-dev/feast) roadmap, this project intends to add Hbase 
+support for Online Store.
```

Comment:
nit: want to move this to the previous line? seems to cut off unnaturally

Reply:
Done!